src/FluentAvalonia/UI/Controls/IconElement/Symbol.cs

```diff
@@ -116,7 +100,7 @@ public enum Symbol
     CloudSyncCompleteFilled = 0xF8044,
     CloudSyncFilled = 0xF8045,
     Code = 0xE943,
-    CodeFilled = 0xF8047,
+    CodeHTML = 0xF8134,
```

Comment:
I know you already merged and this is a nit.

But usually it would be `CodeHtml` following standard/modern .net naming conventions. Just letting you know as this is API related and may not be changeable easily in the future.